org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,158 @@
+{
+    "app-id" : "org.arrl.trustedqsl",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Locale" ],
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
+	"--filesystem=home"
+     ],
+    "modules" : [
+        {
+            "name" : "openssl",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "./Configure --prefix=/app && make depend && make build_libs && make install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.openssl.org/source/openssl-3.1.0.tar.gz",
+                    "sha256" : "aaa925ad9828745c4cad9d9efeb273deca820f2cdcf2c3ac7d7c1212b7c497b4"
+                }
+            ],
+	    "cleanup" : [
+		"/bin/c_rehash",
+		"/bin/openssl",
+		"/lib64/libcrypto.a",
+		"/lib64/libssl.a",
+		"/share/doc/openssl/*",
+		"/share/doc/openssl",
+		"/share/man/man1/*",
+		"/share/man/man3/*",
+		"/share/man/man5/*",
+		"/share/man/man7/*",
+		"/share/man/man1",
+		"/share/man/man3",
+		"/share/man/man5",
+		"/share/man/man7",
+		"/include/openssl/*",
+		"/include/openssl",
+		"/lib64/pkgconfig/*",
+		"/lib64/pkgconfig"
+	    ]
+        },
+        {
+            "name" : "lmdb",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "cd libraries/liblmdb/ && make prefix=/app install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/LMDB/lmdb/archive/LMDB_0.9.23.tar.gz",
+                    "sha256" : "abf42e91f046787ed642d9eb21812a5c473f3ba5854124484d16eadbe0aa9c81"
+                }
+            ],
+	    "cleanup" : [
+		"/include/lmdb.h",
+		"/lib/liblmdb.a",
+		"/share/man/man1/*",
+		"/share/man/man1",
+		"/bin/mdb_copy",
+		"/bin/mdb_dump",
+		"/bin/mdb_load",
+		"/bin/mdb_stat"
+	    ]
+        },
+        {
+            "name" : "curl",
```

Comment:
curl is in the the runtime.

Reply:
Curl *development* is not in the runtime.

Reply:
not package need to build curl.